great_expectations/validation_operators/validation_operators.py

```diff
@@ -27,6 +31,13 @@ class ValidationOperator(object):
     of validation operator classes that will be the descendants of this base class.
     """
 
+    def __init__(self) -> None:
+        self._validation_operator_config = None
+
+    @property
+    def validation_operator_config(self):
```

Comment:
Suggestion to rename this to `checkpoint_config`

Reply:
I think Eugene was going to go over semantic uses of validation_operator vs checkpoint